external/anomaly/tools/sample.py

```diff
@@ -0,0 +1,278 @@
+import argparse
```

Comment:
Looks like this script is not formatted via git-commit. CI would therefore fail. Would be good to apply style-check to the changes